.gitpod.Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM gitpod/workspace-full as base
+
+USER gitpod
+
+
+RUN sudo apt-get update -y && sudo apt-get install -y \
+    # common stuff
+    git \
+    wget \
+    gnupg \
+    apt-transport-https \
+    ca-certificates \
+    apt-utils \
+    curl \
+    jq
+
+# ========================================================================================================
+# Update repository signing keys
+# --------------------------------------------------------------------------------------------------------
+# Hyperledger
+RUN sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9692C00E657DDE61 && \
+    # Sovrin
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
+# ========================================================================================================
+
+# Plenum
+#  - https://github.com/hyperledger/indy-plenum/issues/1546
+#  - Needed to pick up rocksdb=5.8.8
+RUN sudo add-apt-repository 'deb  https://hyperledger.jfrog.io/artifactory/indy focal dev' && \
+    sudo add-apt-repository 'deb http://security.ubuntu.com/ubuntu bionic-security main' && \
+    sudo add-apt-repository 'deb https://repo.sovrin.org/deb bionic master'  && \
+    sudo add-apt-repository 'deb https://repo.sovrin.org/sdk/deb bionic master' 
+
+# Sovrin
+RUN sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88 
```

Comment:
Remove duplicate `apt-key` call for Sovrin.  The initial call is performed on line 23.